haskell-indexer-pipeline-ghckythe/src/Language/Haskell/Indexer/Pipeline/GhcKythe.hs

```diff
@@ -36,15 +37,23 @@ import Language.Haskell.Indexer.Frontend.Kythe (toKythe)
 
 import qualified Language.Kythe.Schema.Raw as Raw
 
+import System.IO
+
 ghcToKythe
     :: MVar () -> GhcArgs -> AnalysisOptions
     -> Raw.VName
-    -> ([Raw.Entry] -> IO ())
+    -> (Handle -> [Raw.Entry] -> IO ())
```

Comment:
does the handle change on each invocation of the sink?

Reply:
Yes, depending on what `-o` is set to.

Reply:
So IIUC the stdout will get replaced with plumbing from the output flag here.

Related to the pipe output commented in detail above, I wonder if the opening/closing behavior needs flexibility (for example, we don't want to close/reopen the same output pipe for each module. I don't think pipes support that. Or stdout.)